forge-core/src/main/java/forge/deck/DeckFormat.java

```diff
@@ -204,17 +205,19 @@ public String getDeckConformanceProblem(Deck deck) {
 
         int min = getMainRange().getMinimum();
         int max = getMainRange().getMaximum();
-        boolean noBasicLands = false;
+        // boolean noBasicLands = false;
 
         // Adjust minimum base on number of Advantageous Proclamation or similar cards
         CardPool conspiracies = deck.get(DeckSection.Conspiracy);
         if (conspiracies != null) {
             min -= (5 * conspiracies.countByName(ADVPROCLAMATION, false));
-            noBasicLands = conspiracies.countByName(SOVREALM, false) > 0;
+            // noBasicLands = conspiracies.countByName(SOVREALM, false) > 0;
+            conspiracies.countByName(SOVREALM, false);
```

Comment:
Whats this change about?

Reply:
Removing warnings.  I left the call to countByName because I didn't want to disrupt the code by removing it entirely, but the noBasicLands boolean is being assigned, but never used (throwing a warning).

Reply:
CountByName doesn't' really do anything except return a number. While I agree assigning to noBasicLands doesn't do anything, it at least is a place holder for someone willing to add logic for decks that don't need to have basics in them. Removing that assignment just removes all of the context of this code, and it feels worse because of that change.

Reply:
With code that I commit, I refuse to leave any warnings in it, so I will comment the call and boolean initialization lines.

Reply:
As this is now outdated, resolving.